hbase-shell/src/main/ruby/shell/commands/deleteall.rb

```diff
@@ -30,12 +30,14 @@ def help
   hbase> deleteall 't1', 'r1'
   hbase> deleteall 't1', 'r1', 'c1'
   hbase> deleteall 't1', 'r1', 'c1', ts1
+  hbase> deleteall 't1', 'r1', '', ts1        //'' means not specify column
```

Comment:
nit: better rephrase comment as: `//'' means no specific column`

Reply:
Maybe worth also explain that this command deletes all cells in the row which timestamp is lower than the one specified in the command.

Reply:
fix, thanks for your review.